src/isBlank.js

```diff
@@ -0,0 +1,39 @@
+import { isEmpty, anyPass, both, test } from 'ramda';
+
+import isString from './isString';
+import isUndefined from './isUndefined';
+import isFalse from './isFalse';
+/**
+ * Checks if input value is `Blank`.
+ *
+ * @func isBlank
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/3.1.0|v3.1.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html|Differences Between #nil?, #empty?, #blank?, and #present?}
+ * @example
+ *
+ * RA.isBlank(''); //=> true
+ * RA.isBlank('   '); //=> true
+ * RA.isBlank('\t\n'); //=> true
+ * RA.isBlank('value'); //=> false
+ * RA.isBlank({}); //=> true
+ * RA.isBlank({ foo: 'foo' }); //=> false
+ * RA.isBlank([]); //=> true
+ * RA.isBlank([1, 2, 3]); //=> false
+ * RA.isBlank(null); //=> false
+ * RA.isBlank(undefined); //=> false
+ * RA.isBlank(true); //=> false
+ * RA.isBlank(false); //=> true
+ */
+const isBlank = anyPass([
+  isFalse,
+  isUndefined,
+  isEmpty,
+  both(isString, test(/^\s+$/gm)),
```

Comment:
I've tested that R.test is safe for all inputs. So this predicated can be further reduced to following:
```suggestion
  test(/^\s+$/gm)
```

I'll try to finish the review during tommorow.

Reply:
sounds good .. i'll update that condition 😄